incident-response-plan.md

```diff
@@ -0,0 +1,153 @@
+# Incident Response Plan
+
+## Purpose  
+This document outlines the process for handling security-impacting incidents affecting one or more projects in the Foundation’s ecosystem. Incidents may include platform changes with security implications, account compromise, or other security events that require coordinated response.
```

Comment:
Suggested rewording:

This document outlines the process for handling security incidents affecting any OpenJS project.